project-base/storefront/components/Pages/ProductDetail/ProductDetailTabs/ProductDetailTabs.tsx

```diff
@@ -40,30 +40,78 @@ export const ProductDetailTabs: FC<ProductDetailTabsProps> = ({ description, par
+                                    .sort((a, b) => a.name.localeCompare(b.name))
+                                    .map((parameter) => (
+                                        <Row
+                                            key={parameter.uuid}
+                                            className="bg-tableBackground odd:bg-tableBackgroundContrast border-none"
+                                        >
+                                            <Cell className="py-2 text-left text-sm font-bold uppercase leading-5">
+                                                {parameter.name}
+                                            </Cell>
+                                            <Cell className="py-2 text-right text-sm leading-5">
+                                                {parameter.values.map((value, index) =>
+                                                    formatParameterValue(
+                                                        value.text + (parameter.unit?.name ? ` (${parameter.unit.name})` : ''),
+                                                        index
+                                                    )
+                                                )}
+                                            </Cell>
+                                        </Row>
+                                    ))}
+                            </Table>
+                        </div>
+                    )}
 
-                                <Cell className="py-2 text-right text-sm leading-5">
-                                    {parameter.values.map((value, index) =>
-                                        formatParameterValue(
-                                            value.text + (parameter.unit?.name ? ` (${parameter.unit.name})` : ''),
-                                            index,
-                                        ),
-                                    )}
-                                </Cell>
-                            </Row>
-                        ))}
-                    </Table>
+                    {Object.entries(
+                        parameters.reduce((acc, parameter) => {
+                            if (parameter.group) {
+                                if (!acc[parameter.group]) {
+                                    acc[parameter.group] = [];
+                                }
+                                acc[parameter.group].push(parameter);
+                            }
+                            return acc;
+                        }, {})
+                    ).map(([groupName, groupParameters]) => (
+                        <div key={groupName}>
+                            <h2 className="text-lg font-bold max-w-screen-lg mx-auto my-4">{groupName}</h2>
+                            <Table className="border-0 p-0 max-w-screen-lg mx-auto">
+                                {groupParameters
+                                    .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
This sorting can also be moved into variable and named as `sortedGroupParameters` for example.